perfkitbenchmarker/publisher.py

```diff
@@ -185,6 +185,9 @@ def AddMetadata(self, metadata, benchmark_spec):
           for key, value in data_disk.metadata.iteritems():
             metadata[name_prefix + 'data_disk_0_' + key] = value
```

Comment:
Thanks for pointing that out! I wrote that code. Right now `key` will always be a string, but I see your point that it is more robust to use `%s`-conversion. However, that's a separate PR. If you write it, I will LGTM :)